### ISSUE TITLE
Allow encryption/decryption without files

### ISSUE THREAD

**Issue by xmakina** [stx-labs/stacks.js#269]
In some use cases, some data may need encrypting/decrypting without involving a complete file. An API endpoint which will run en/decryption on the Blockstack core (avoiding the many issues with client-side encryption) would be very helpful.

**Comment by larrysalibra** [comment]
This would be easy to add.

We'd just to expose these 3 lines as a public api call: 
https://github.com/blockstack/blockstack.js/blob/3d310a6effd074a5495e5325adec6f320023295b/src/storage/index.js#L39
https://github.com/blockstack/blockstack.js/blob/3d310a6effd074a5495e5325adec6f320023295b/src/storage/index.js#L40
https://github.com/blockstack/blockstack.js/blob/3d310a6effd074a5495e5325adec6f320023295b/src/storage/index.js#L41

**Comment by nivas8292** [comment]
Can i pick this up?

**Comment by larrysalibra** [comment]
We have an open pull request for this #274 - it needs test coverage before we can merge it in. @nivas8292 are you still up for doing that?

**Comment by nikkolasg** [comment]
I tried to take on @nivas8292 's work and wrote a test for the new function (`encryptContent`). Unfortunately, the `loadUserData()` call fails with an invalid JSON descriptor (seems like there's no user signed in in the tests ?). Searching a bit, I also realized that the `storage/putFile` function never gets tested with the encryption functionality. When putting the option to encrypt to true, the same error occurs (because it uses the same function `loadUserData`). 
I searched a while in the tests but I could not find any related test to `loadUserData`:
+ Any reasons in particular why there's no tests for encrypted files ? 
+ Is it possible to make `loadUserData` suceeds in the regular tests ? Any hints how ? I imagine with `proxrequire` but I'm not sure how yet...
